app/core/config.py

```diff
@@ -10,6 +10,16 @@
 
 
 class Settings(BaseSettings):
+    """Settings is class to hold all the configuration information about the server
+
+    Parameters
```

Comment:
This part is not needed. `BaseSettings` is just the parent class and shouldn't be documented as a parameter.